Firebase/Core/Private/FIRBundleUtil.h

```diff
@@ -45,8 +45,8 @@
 + (NSArray *)relevantURLSchemes;
 
 /**
- * Checks if the bundle identifier exists in the given bundles.
+ * Checks the given bundles to see of them is a prefix of the given identifier.
```

Comment:
Maybe "Checks if any of the given bundles have a matching bundle identifier prefix (removing extension suffixes)."?